django_addanother/templates/django_addanother/related_widget_wrapper.html

```diff
@@ -5,15 +5,15 @@
     {% if edit_related_url %}
     <a class="related-widget-wrapper-link change-related" id="change_id_{{ name }}"
         href=""
-        data-href-template="{{ edit_related_url }}?{{ url_params }}"
+        data-href-template="{{ edit_related_url }}{{ edit_params_separator }}{{ url_params }}"
```

Comment:
This feels a bit messy; how about we do all of the URL preparation stuff in Python and profit from `urlparse`? (Keep in mind to use `six` in that case)